src/app.js

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env node
+// shebang
+
+// const { createCanvas, loadImage, Image } = require('canvas')
+const canvas = require('canvas');
+
+const fs = require('fs');
+// we need fs to read the image & redraw it onto canvas
+
+const yargs = require('yargs');
+// yargs allows us to configure for arguments 
+// we need to accept multiple arguments for business info (5x mandatory & 3x optional) and a JSON object (mandatory)  
+// we need to accept 1 x mandatory argument for JSON file path 
+// we need to accept 1 x optional argument for Locale tag (language) 
+
+const options = yargs
+.usage("Usage: -n <name>")
+.option("o", {alias: "order", describe: "JSON file path", type: "string", demandOption: true})
+.option("n", {alias: "name", describe: "Your name - 20 char limit", type: "string", demandOption: true})
+.option("a", {alias: "address1", describe: "Number, Street, City - 53 char limit", type: "string", demandOption: true})
+.option("b", {alias: "address2", describe: "Country & UK postcode - 53 char limit", type: "string", demandOption: true})
+.option("v", {alias: "vat", describe: "VAT number", type: "string", demandOption: true})
+.option("i", {alias: "image", describe: "url to image for logo", type: "string", demandOption: true})
+.option("t", {alias: "telephone", describe: "Telephone number", type: "number", demandOption: false })
+.option("w", {alias: "website", describe: "Website - 48 char limit", type: "string", demandOption: false})
+.option("t", {alias: "topMessage", describe: "Top Message - 30 char limit", type: "string", demandOption: false})
+.option("l", {alias: "locale", describe: "locale tag", type: "string", demandOption: false})
+.choices("l", ["en", "de", "zh"])
+.argv;
+ 
+// TOO: The Order info: Class that takes JSON file path - do we need to use fs function fs.createReadFileStream
+
+// check
+console.log(`Business name: ${options.name}, Address: ${options.address1 + " "  + options.address2}, VAT: ${options.vat}, Logo/Image URL: ${options.image}`);
+
+// validates mandatory CLI arguments with char limit
+const validate = (arg, limit = 53) => {
+  try {
+    if (options[arg].length > limit) {
+      throw new Error( `${arg} exceeds ${limit} character limit. Valid input required.` );
+      // throw new Error exposes an error event with two params name & message. It also terminates further execution. 
+    }
+  } catch (err) {
+      console.log(err.name, err.message);
+  }
+}
+
+validate("name", 20); 
+validate("address1");
+validate("address2");
+validate("vat");
+validate("telephone");
+validate("website");
+validate("topMessage", 30);
+
+// resize logo
+// gets original dimensions of image
+let oWidth; 
+let oHeight; 
+
+const img = new canvas.Image();
```

Comment:
This looks really good :) as you're creating images already, I think it'd be worth setting up an end-to-end test that generates an image sooner rather than later. This will really help debugging issues and give you confidence that the application is doing what it's supposed to.